design/database.d.ts

```diff
@@ -37,7 +37,7 @@ export interface event extends dbEntry {
     
     volunteerNames: runner[]
 
-    stats: {
+    counts: {
         no2k: number
```

Comment:
As part of my request, could you also rename the properties of `count` to something a bit more readable, and do not include `no`? such as `2k` `5k` `runners` `volunteers`? The "number" part of the name is implied by being in the "count" property.